nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml

```diff
@@ -196,10 +195,11 @@ local_costmap:
           raytrace_min_range: 0.0
           obstacle_max_range: 2.5
           obstacle_min_range: 0.0
+      static_layer:
+        plugin: "nav2_costmap_2d::StaticLayer"
+        map_subscribe_transient_local: True
       always_send_full_costmap: True
 
-# GPS WPF CHANGE: Remove static layer
```

Comment:
Ditto

Reply:
done in https://github.com/ros-navigation/navigation2_tutorials/pull/115/commits/266b24b3b8c651614f2231178eed256e68a2fa27